### PR TITLE
5.1.0

### PR DESCRIPTION
**Pull request by mighttymike**

### CODE REVIEW

File: packages/flutter_timeline/CHANGELOG.md
Comment (Gorter-dev) [inline packages/flutter_timeline/CHANGELOG.md:1]: Make this an actual symbolic link

File: packages/flutter_timeline/README.md
Comment (Gorter-dev) [inline packages/flutter_timeline/README.md:1]: Make this an actual symbolic link

File: packages/flutter_timeline/LICENSE
Comment (Gorter-dev) [inline packages/flutter_timeline/LICENSE:1]: Make this an actual symbolic link

File: packages/flutter_timeline_firebase/CHANGELOG.md
Comment (Gorter-dev) [inline packages/flutter_timeline_firebase/CHANGELOG.md:1]: Make this an actual symbolic link

File: packages/flutter_timeline_firebase/LICENSE
Comment (Gorter-dev) [inline packages/flutter_timeline_firebase/LICENSE:1]: Make this an actual symbolic link

File: packages/flutter_timeline_firebase/README.md
Comment (Gorter-dev) [inline packages/flutter_timeline_firebase/README.md:1]: Make this an actual symbolic link

File: packages/flutter_timeline_interface/CHANGELOG.md
Comment (Gorter-dev) [inline packages/flutter_timeline_interface/CHANGELOG.md:1]: Make this an actual symbolic link

File: packages/flutter_timeline_interface/LICENSE
Comment (Gorter-dev) [inline packages/flutter_timeline_interface/LICENSE:1]: Make this an actual symbolic link

File: packages/flutter_timeline_interface/README.md
Comment (Gorter-dev) [inline packages/flutter_timeline_interface/README.md:1]: Make this an actual symbolic link

File: packages/flutter_timeline_view/CHANGELOG.md
Comment (Gorter-dev) [inline packages/flutter_timeline_view/CHANGELOG.md:1]: Make this an actual symbolic link

File: packages/flutter_timeline_view/LICENSE
Comment (Gorter-dev) [inline packages/flutter_timeline_view/LICENSE:1]: Make this an actual symbolic link

File: packages/flutter_timeline_view/README.md
Comment (Gorter-dev) [inline packages/flutter_timeline_view/README.md:1]: Make this an actual symbolic link

**Review by Gorter-dev — CHANGES_REQUESTED** [review]

File: .github/workflows/melos-component-ci.yml
Comment (freekvandeven) [inline .github/workflows/melos-component-ci.yml:1]: i prefer you not specify it unless the newest version is breaking the package

**Review by freekvandeven — CHANGES_REQUESTED** [review]

**Review by freekvandeven — APPROVED** [review]

File: packages/flutter_timeline_view/pubspec.yaml [hunk 1]
```diff
@@ -4,31 +4,26 @@
 
 name: flutter_timeline_view
 description: Visual elements of the Flutter Timeline Component
-version: 4.1.0
-
-publish_to: none
+version: 5.1.0
```
Comment (freekvandeven) [inline packages/flutter_timeline_view/pubspec.yaml:7]: you should make it 5.0.0

**Review by freekvandeven — CHANGES_REQUESTED** [review]

**Review by Gorter-dev — APPROVED** [review]

**Pull request merged**